slides/src/content/unit-testing.md

```diff
@@ -498,74 +499,62 @@ export class TodoService {
 ```
 
 ---
-<!-- .slide: id="unit-testing-creating-and-injecting-mock-object-1" -->
-## Creating and Injecting the Mock Object
-
-- The backend returns data that looks like this:
-
-```json
-{
-  "todos": [
-    {
-      "done": true,
-      "id": 1,
-      "label": "Wake up in the morning"
-    },
-    {
-      "done": true,
-      "id": 2,
-      "label": "Grab glasses, go out the door, hit the city"
-    }
-  ]
-}
-```
+<!-- .slide: id="unit-testing-light-http-mock" -->
+## Providing a Light HTTP Mock
 
----
-<!-- .slide: id="unit-testing-creating-and-inject-mock-object-2" -->
-## Creating and Injecting the Mock Object
-
-- We can create a light mock of the `Http` service:
+- A light mock can be used in place of the actual Http service
```

Comment:
1.  back ticks around Http
2.  What's a "light mock"?  Light as in "lightweight"?